src/test/java/org/broadinstitute/consent/http/db/LibraryCardDAOTest.java

```diff
@@ -189,6 +193,54 @@ public void testDeleteLibraryCardByUserId() {
     assertNull(libraryCardDAO.findLibraryCardById(card.getId()));
   }
 
+  @Test
+  public void testCreateLibraryCardDaaAssociation() {
```

Comment:
The following tests don't have any assertions in them, we should likely try to check on what was created/deleted

Reply:
Secondarily, we should try to clean up our test classes for JUnit5 best practices. That would mean removing the public modifiers from the class and all test methods, as well as adding `@ExtendWith(MockitoExtension.class)` as a class annotation. Those details help us keep tests cleaner and more efficient.